ecology.yaml

```diff
@@ -775,3 +775,7 @@ tools:
   - name: ab1_fastq_converter
     owner: ecology
     tool_panel_section_label: 'Convert Formats'
+    
+  - name: aligned_to_consensus
```

Comment:
Are you sure the tool is named like that in the TS? It is not available?

Reply:
It was right actually